invenio_queues/ext.py

```diff
@@ -71,12 +70,16 @@ class InvenioQueues(object):
     """Invenio-Queues extension."""
 
     def __init__(self, app=None, **kwargs):
-        """Extension initialization."""
+        """Initialize extension."""
         if app:
             self.init_app(app, **kwargs)
 
+    def __getattr__(self, name):
+        """Proxy to state object."""
+        return getattr(self._state, name, None)
+
     def init_app(self, app, entry_point_group='invenio_queues.queues'):
-        """Flask application initialization."""
+        """Initialize application."""
         self.init_config(app)
         app.extensions['invenio-queues'] = _InvenioQueuesState(
```

Comment:
It's better to do this, so that `__getattr__` actually works: 

```python
self._state = _InvenioQueuesState(...)
app.extensions['invenio-queues'] = self._state
```